xcell/mappers/mapper_SPT.py

```diff
@@ -1,4 +1,5 @@
 from .mapper_Planck_base import MapperPlanckBase
+from .utils import rotate_map
```

Comment:
I just realised the SPT maps are provided in equatorial (not Galactic) coordinates